integrations/azure-devops/.port/resources/port-app-config.yaml

```diff
@@ -35,3 +38,20 @@ resources:
           blueprint: '"service"'
           properties:
             workItemLinking: '.isEnabled and .isBlocking'
+
+  - kind: project
+    selector:
+      query: 'true'
+      defaultTeam: "true"
+    port:
+      entity:
+        mappings:
+          identifier: '.id'
+          blueprint: '"project"'
+          title: .name
+          properties:
+            state: '.state'
+            revision: '.revision'
+            visibility": '.visibility'
+            defaultTeam: '.defaultTeam.name'
+            link: '.url | gsub("_apis/projects/"; "")'
```

Comment:
the order of the mapping should be in the order of hierarchy, meaning : project first and then what that is connected to it repository